schemas/container.yml

```diff
@@ -75,6 +75,12 @@
       normalize:
         - array
 
+    - name: image.hash
```

Comment:
For container images, is there one hash type used consistently, like `sha-256`?

If so, I wonder if we should include the type in the name. It's a convention used elsewhere in ECS (for example, [`tls.client.hash.sha256`](https://www.elastic.co/guide/en/ecs/current/ecs-tls.html#field-tls-client-hash-sha256), and this approach allows for adding other hash types later, if necessary.

Reply:
For sure; it's usually sha-256. Maybe that was after your comments, but I changed this last week and included an example and description.  This value would look like this `sha256:f8fefc80e3273dc756f288a63945820d6476ad64883892c771b5e2ece6bf1b26`.

That said, I don't see any reason to deviate from the ECS convention. I'll change it.